### PR TITLE
{bio}[GCCcore/11.2.0] Porechop-0.2.4 w/Python/3.9.6

### PR DESCRIPTION
**Pull request by fizwit**
(created using `eb --new-pr`)

### CODE REVIEW

**Review by verdurin — CHANGES_REQUESTED** [review]

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/3a36111e30273780e0c645b0d6751ed3 for a full test report.

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/p/Porechop/Porechop-0.2.4-GCCcore-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ it is treated as chimeric and chopped into separate reads. Porechop performs thorough alignments
+ to effectively find adapters, even at low sequence identity"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+
+source_urls = ['https://github.com/rrwick/Porechop/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['44b499157d933be43f702cec198d1d693dcb9276e3c545669be63c2612493299']
+
+dependencies = [
```
[the remark this answers is not in the document]
Comment (fizwit) [reply easybuild/easyconfigs/p/Porechop/Porechop-0.2.4-GCCcore-11.2.0.eb:27]: done

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @fizwit!

**Pull request merged**